docker-compose.yml

```diff
@@ -101,6 +101,12 @@ services:
     env_file:
       - ./env.list
     restart: always
+    # PRODUCTION NOTE: For Linux production environments, enable the following:
+    # sysctls:
+    #   - vm.overcommit_memory=1
+    # This prevents Redis memory overcommit warnings and potential issues.
+    # For local development on macOS/Windows, keep this commented out
+    # to avoid Docker errors.
```

Comment:
docker doesnt allow this sysctl to be set in this way. I think we should remove these comments so we don't mislead anyone